### PR TITLE
Fix/solver workflow typos

### PR DESCRIPTION
**Pull request by goern**

### CODE REVIEW

File: reconcile.sh [hunk 1]
```diff
@@ -2,9 +2,15 @@
...
 
-for w in $( oc get workflows -o=jsonpath='{.items[?(@.status.phase=="Error")].metadata.name}' ); do
-    argo retry $w
-    sleep 7
+while :
+do
+    for w in $(oc get workflows -o=jsonpath='{.items[?(@.status.phase=="Error")].metadata.name}' ); do
+        argo retry $w
+        sleep 7
+        oc delete workflow $(oc get workflows -o=jsonpath='{.items[?(@.status.phase=="Succeeded")].metadata.name}')
```
Comment (CermakM) [inline reconcile.sh:10]: By this, we're saying that the whole logic is encapsulated in the Workflow and there is no other component interacting with it that is not part of the Workflow. Is that a valid assumption?

Comment (goern) [reply reconcile.sh:10]: for a solver workflow... i think that is true

Comment (goern) [reply reconcile.sh:10]: @fridex

Comment (fridex) [reply reconcile.sh:10]: Once we have data on ceph and the sync to the graph is done, we don't need openshift objects anymore.

**Review by CermakM — APPROVED** [review]

**Pull request merged**